src/app_engine/constants.py

```diff
@@ -15,7 +16,13 @@
 TURN_URL_TEMPLATE = '%s/turn?username=%s&key=%s'
 CEOD_KEY = '4080218913'
 
-WSS_HOST_PORT_PAIR = 'apprtc-ws.webrtc.org:443'
+WSS_HOST_PORT_PAIRS = ['apprtc-ws.webrtc.org:443', 'apprtc-ws-2.webrtc.org:443']
+WSS_HOST_STATUS_LIST_KEY = 'wss_host_status_list'
+
+WSS_HOST_INSTANCE_KEY = 'instance'
+WSS_HOST_STATUS_KEY = 'status'
+WSS_HOST_STATUS_CODE_KEY = 'status_code'
```

Comment:
Prior to my change, the http request would just return the error_message if there was one, and set the http return status. Now it returns a json doc with the instance name, error message, http status code and up/down status. If we don't care about returning the error message via the http call, then yes, we can.